envoy-control-tests/src/main/resources/envoy/envoy2_auth_config.yaml

```diff
@@ -23,6 +23,7 @@ node:
     use_remote_address: true
     access_log_enabled: false
     add_upstream_external_address_header: true
+    has_static_secrets_defined: true
```

Comment:
I can't find usage of this property.

Reply:
Good catch! I've added too much to this part of the pr, will remove it and add it when it's used.